course_catalog/etl/podcast.py

```diff
@@ -0,0 +1,195 @@
+"""podcast ETL"""
+
+import logging
+from datetime import datetime
+from django.conf import settings
+import github
+import yaml
+from bs4 import BeautifulSoup as bs
+import requests
+from course_catalog.etl.utils import generate_unique_id
+from course_catalog.constants import PlatformType
+
+CONFIG_FILE_REPO = "mitodl/open-podcast-data"
+CONFIG_FILE_FOLDER = "podcasts"
+
+log = logging.getLogger()
+
+
+def github_podcast_config_files():
+    """
+    Function that returns a list of pyGithub files with youtube config channel data
+
+    Returns:
+        A list of pyGithub contentFile objects
+    """
+    github_client = github.Github()
+    repo = github_client.get_repo(CONFIG_FILE_REPO)
```

Comment:
Can we make `CONFIG_FILE_REPO` and `CONFIG_FILE_FOLDER` into environment variables?